ui/src/types/index.ts

```diff
@@ -100,7 +106,9 @@ export type UserInfoData = {
   webAuthnCreationOptions?: WebAuthnCreationOptions;
   webAuthnRequestOptions?: WebAuthnRequestOptions;
   webAuthnUrl?: string;
-  runtimeFlags?: Record<string, boolean>;
+  runtimeFlags?: {
+    routes_portal?: boolean;
+  } & Record<string, boolean | undefined>;
```

Comment:
I removed this as there is nowhere any other runtimeFlags are being used atm, just `routes_portal` in the frontend.